exir/passes/cache_pos_init_mutable_pass.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from executorch.exir.pass_base import ExportPass
+
+
+class CachePosToInitializedMutableBufferPass(ExportPass):
```

Comment:
Let's make this pass more generic for people who need it in the future. Maybe call it `InitializeMutableBufferPass` that takes in a list of strings that are the names of the buffers that need to be initialized. We find those buffers and then set the node metadata on them. If we don't find the buffer of that name we can raise an Exception.